### ISSUE TITLE
Line breaks handled differently on iOS and Android

### ISSUE THREAD

**Issue by skypanther** [FokkeZB/nl.fokkezb.html2as.widget#3]
To get line breaks in your attributed string, you need to use `\n` on iOS and `<br/>` on Android. It would be nice if this widget normalized that, or supported either, across platforms.

**Comment by FokkeZB** [comment]
I just updated and tested with the example in https://github.com/FokkeZB/ti-html2as and both iOS and Android show the same behaviour for the widget where only `<br />` results in a newline.

Do you have an example that shows otherwise? Didn't have too much time to look at it, so I might overlook something.

**Comment by skypanther** [comment]
It could be because I'm updating the contents of my html2as-ified label in code after a change in state in my app. To get line breaks, I had to use the following.

```
var msg = 'We\'ve <strong>emailed you an invoice</strong>...\n\nUse our secure payment partner....\n\nOnce completed we\'ll send you...';
if (OS_ANDROID) {
    msg = msg.replace(/\n/g, '<br/>');
}
```

**Comment by FokkeZB** [comment]
So after these lines you do `$.theWidget.html = msg` ?

**Comment by skypanther** [comment]
Yes, exactly. Sorry that's buried in a function elsewhere in my code so i didn't include that earlier.

**Comment by FokkeZB** [comment]
I've tried setting it after creation with your HTML but still I see identical behaviour: only `<br />` works, on both platforms. Could you check the version of `ti-html2as` in your project's modules folder? should be 1.4

**Comment by FokkeZB** [comment]
I tried your code (setting HTML after creation) on the widget in the example but I still get identical behaviour (only `<br />` works) on both iOS and Android. Can you verify what ti-html2as module your project has? Should be 1.4
